src/Plugin.InAppBilling/Plugin.InAppBilling.csproj

```diff
@@ -11,7 +11,7 @@
     <Product>$(AssemblyName) ($(TargetFramework))</Product>
     <AssemblyVersion>7.0.0.0</AssemblyVersion>
     <AssemblyFileVersion>7.0.0.0</AssemblyFileVersion>
-    <Version>7.0.0.0</Version>
+    <Version>8.1.4.0-ramsey-beta</Version>
```

Comment:
Version that will be used in the nuget package I have been testing locally. It will also be the version I push up to Azure artifacts to be used in the Purchase library: https://github.com/lampo/in-app-purchase-csharp/pull/33